--- goptical-1.0/goptical_core/configure.ac.orig	2024-09-29 18:32:47.739695176 +0200
+++ goptical-1.0/goptical_core/configure.ac	2024-09-29 19:18:29.941506060 +0200
@@ -143,10 +143,10 @@ AM_CONDITIONAL(GOPTICAL_HAVE_X11, test x
 AC_ARG_ENABLE(plplot, AC_HELP_STRING(--disable-plplot, [Disable use of PlPlot renderer]), GOPTICAL_HAVE_PLPLOT=false)
 
 if test x$GOPTICAL_HAVE_PLPLOT != xfalse ; then
-AT_CHECK_CPPLIB(-lplplotd -lplplotcxxd, plplot/plstream.h, new plstream();, [
+AT_CHECK_CPPLIB(-lplplot -lplplotcxx, plplot/plstream.h, new plstream();, [
   GOPTICAL_HAVE_PLPLOT=true
   AC_DEFINE(GOPTICAL_HAVE_PLPLOT, 1, [plplot rendering library enabled])
-  LIBS="$LIBS -lplplotcxxd -lplplotd "
+  LIBS="$LIBS -lplplotcxx -lplplot "
 ])
 fi
 AM_CONDITIONAL(GOPTICAL_HAVE_PLPLOT, test x$GOPTICAL_HAVE_PLPLOT = xtrue)
--- goptical-1.0/goptical_core/src/io_renderer_plplot.cc.orig	2011-11-09 20:52:18.000000000 +0100
+++ goptical-1.0/goptical_core/src/io_renderer_plplot.cc	2024-09-29 19:24:45.106140285 +0200
@@ -188,7 +188,7 @@ namespace _Goptical {
                                      const Rgb &rgb, bool filled)
     {
       _pls->col0(get_color_id(rgb));
-      _pls->arc(c.x(), c.y(), r, r, 0., 360., filled);
+      _pls->arc(c.x(), c.y(), r, r, 0., 360., 0., filled);
     }
 
     void RendererPlplot::draw_polygon(const Math::Vector2 *array, unsigned int count,
